### PR TITLE
Fix the wrong indentation to pass grammar check

### LINKED ISSUE #13785: Alluxio CSI cannot works from helm-generate.sh
**Alluxio Version:**

Master

**Describe the bug**

yaml files parse fail.

error validating "/Users/mbl/projects/tencent/AlluxioOnKubernetes/multiMaster-embeddedJournal/csi/alluxio-csi-nodeplugin.yaml": error validating data: ValidationError(DaemonSet.spec.template.spec.containers[1].envFrom[0]): unknown field "name" in io.k8s.api.core.v1.EnvFromSource; if you choose to ignore these errors, turn validation off with --validate=false

Also, It would be helpful if Alluxio supply a document to describe the alluxio-csi limitation. Something like

- To use alluxio-csi, we have to get admin role of kubernates cluster?
- alluxio-csi-nodeplugin is a daemonset resource, so it will run in every k8s node, if your cluster have 3000 node, 3000 pod will be scheduled.

**To Reproduce**

```bash
cd integration/kubernetes
helm-generate.sh  multi-embedded
```

**Expected behavior**

All CSI resources are correct and can be applied to k8s cluster.

**Urgency**

Urgent

**Additional context**

No

### PR DESCRIPTION
**Pull request by maobaolong**
Fix #13785

### CODE REVIEW

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks @maobaolong !

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
Please also mention this change in the CHANGELOG
https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/CHANGELOG.md
and https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm-chart/alluxio/Chart.yaml
Thanks!

**Comment by jiacheliu3** [comment]
I will merge this change after the change above is added

**Comment by maobaolong** [comment]
@jiacheliu3 Thanks for your review and suggestion, I upgrade the version and change log of alluxio helm chart,  PTAL.

File: integration/kubernetes/helm-chart/alluxio/Chart.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 name: alluxio
 apiVersion: v1
 description: Open source data orchestration for analytics and machine learning in any cloud.
-version: 0.6.22
+version: 0.6.23
```
Comment (jiacheliu3) [inline integration/kubernetes/helm-chart/alluxio/Chart.yaml:15]: @maobaolong This should be 0.6.24?

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
@maobaolong One last minor typo to change. The version you see is 0.6.22 just because that guy forgot to update it to 0.6.23

File: integration/kubernetes/helm-chart/alluxio/Chart.yaml (hunk 1 above)
Comment (maobaolong) [reply integration/kubernetes/helm-chart/alluxio/Chart.yaml]: Yeah, exactly.

**Comment by maobaolong** [comment]
@jiacheliu3 Thanks for the quick review, PTAL.

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks @maobaolong ! Will merge on tests all pass

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**